mantidimaging/gui/windows/operations/view.py

```diff
@@ -231,6 +233,24 @@ def roi_visualiser(self, roi_field):
         window.setCentralWidget(self.roi_view)
         self.roi_view.setWindowTitle("Select ROI for operation")
 
+        def toggle_average_images(images_):
```

Comment:
Can we move this into `windows/roi_selector`? It's getting pretty big to be a function in the Operations window

Reply:
I think if we need to add anything else to the ROI selector, then sure, but right now I was trying to keep it as a "Dialog" similar to QMessageBox